src/racing/render_courses.c

```diff
@@ -113,71 +113,87 @@ void load_surface_map(uintptr_t addr, struct UnkStruct_800DC5EC *arg1) {
     }
     arg1->playerDirection = var_a3;
 
-    if (D_80152300[temp_a2 - camera1] == 1) {
```

Comment:
Is this deletion meant to be part of the PR?

Reply:
Naa, I added an enhancement I need to remove.